Installer/installer.iss

```diff
@@ -1,4 +1,4 @@
-#define public Version "0.6.1"

```

Comment:
This seems unnecessary.

Reply:
Thank You for Your Suggestion. I am going to change this file.